src/index.ts

```diff
@@ -353,6 +353,7 @@ function main() {
                             answers.dockerTag = 'testing';
                         } else if (program.dockerTagOverride) {
                             answers.dockerTag = program.dockerTagOverride;
+                            answers.version = 'master';
```

Comment:
Should this be master? What is getting set to before?

Reply:
In this if condition only dockerTagOverride is provided as an option, we also need to set the overrideVersion explicitly as it does not pick the default value.

Reply:
This basically fixes a bug that existed in the prod version. In this scenario version was blank and it would fail with invalid version.